bld/src/proxyd/log.config

```diff
@@ -5,6 +5,7 @@
 strict init=true
 buffer min=1024
 buffer max=0
+file perms=644
```

Comment:
We had an issue that we could not scp the log files from the shared volume used by the proxy docker container.  While this is a sample config, when using command line to set the log file, 604 is used.  Maybe it makes sense to to allow the default 600 to be overridden via command line.